### PR TITLE
hmmalign version bump. standardised output format, ported to nf-test

### PR DESCRIPTION
**Pull request by vagkaratzas**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/hmmer/hmmalign/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    test("test-hmmer-hmmalign") {
+
+        when {
+            process {
+                """
+                input[0] = [
+				    [ id: 'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'delete_me/hmmer/e_coli_k12_16s.fna.gz', checkIfExists: true)
+				]
+				input[1] = file(params.modules_testdata_base_path + 'delete_me/hmmer/bac.16S_rRNA.hmm.gz', checkIfExists: true)
```
Comment (jfy133) [inline modules/nf-core/hmmer/hmmalign/tests/main.nf.test:22]: Funky idnetitation here

**Review by jfy133 — APPROVED** [review]
LGTM!

Only thing would be are the other hmm tools also on 3.4?

Might be worht bumping them all so the yare all in sync? But I dunno how many hmmer modules there are currently :grimacing:

File: modules/nf-core/hmmer/hmmalign/tests/main.nf.test (hunk 1 above)
Comment (vagkaratzas) [reply modules/nf-core/hmmer/hmmalign/tests/main.nf.test]: Pushed 'em in!

**Comment by vagkaratzas** [comment]
> LGTM once tests pass!
> 
> Only thing would be are the other hmm tools also on 3.4?
> 
> Might be worht bumping them all so the yare all in sync? But I dunno how many hmmer modules there are currently 😬

Bumped hmmbuild version as well!

**Pull request closed without merging**